apps/email/locales/email.en-US.properties

```diff
@@ -660,6 +660,21 @@ drawer-settings-link=Settings
 # font size to fit longer text.
 folder-last-synced-label=Last sync:
 
+# LOCALIZATION NOTE(refresh-button): For the message-list, a label for a button
+# that refreshes the list of messages in the folder. The following string is
+# spoken by screen readers and not shown on the screen.
+refresh-button.ariaLabel=Refresh
```

Comment:
I would be fine if these started with `messages-` to indicate they are for the message list, but do not feel too strongly about it.